README.md

```diff
@@ -237,35 +208,34 @@ Example:
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
+Feel free to check the issues page.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- SUPPORT -->
 
 ## ⭐️ Show your support <a name="support"></a>
 
-> Write a message to encourage readers to support your project
 
-If you like this project...
+If you like this project, please give it a ⭐️!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- ACKNOWLEDGEMENTS -->
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
 
 I would like to thank...
+<p> * Microverse for providing the opportunity to learn Git and GitHub in a collaborative environment.</p>
+<p> * GitHub Docs for providing a wealth of information on Git and GitHub.</p>
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
```

Comment:
- [ **OPTIONAL** ] As it is optional to have the `## ❓ FAQ ` section in your README, I think it would be better if you customize with appropriate content information or completely remove it from your documentation to make it looks more professional